zigbee/Zigbee-Introduction-of-Zigbee-Basic-CN.md

```diff
@@ -0,0 +1,386 @@
+[English](Zigbee-Introduction-of-Zigbee-Basic.md) | 中文
+
+<details>
+<summary><font size=5>目录</font> </summary>  
+&nbsp;
+
+- [1. 概述](#1-概述)
+  - [1.1. 什么是Zigbee](#11-什么是Zigbee)
+  - [1.2. Zigbee的历史](#12-Zigbee的历史)
+  - [1.3. Zigbee联盟](#13-Zigbee联盟)
+  - [1.4. 协议概述](#14-协议概述)
+- [2. 物理层](#2-物理层)
+  - [2.1. 调制](#21-调制)
+  - [2.2. 输出功率](#22-输出功率)
+  - [2.3. 数据速率](#23-数据速率)
+  - [2.4. 开阔范围](#24-开阔范围)
+  - [2.5. 摘要](#25-摘要)
+- [3. 媒体访问控制(MAC)层](#3-媒体访问控制-(MAC)-层)
+  - [3.1. CSMA-CA](#31-csma-ca)
+  - [3.2. 确认](#32-确认)
+  - [3.3. MAC帧](#33-MAC帧)
+- [4. 网络层](#4-网络层)
+  - [4.1. 设备类型](#41-设备类型)
+  - [4.2. 网络地址](#42-网络地址)
+    - [4.2.1. PAN ID](#421-pan-id)
+    - [4.2.2. 扩展 PAN ID](#422-扩展-pan-id)
+  - [4.3. 节点地址](#43-节点地址)
+- [5. 应用层](#5-应用层)
+  - [5.1. 终端](#51-终端)
+  - [5.2. Clusters](#52-Clusters)
+  - [5.3. 例子](#53-例子)
+- [6. 安全性](#6-安全性)
+  - [6.1. 网络层安全](#61-网络层安全)
+    - [6.1.1. 总览](#611-总览)
+    - [6.1.2. 逐跳安全](#612-逐跳安全)
+    - [6.1.3. 网络密钥](#613-网络密钥)
+    - [6.1.4. 帧计数器](#614-帧计数器)
+  - [6.2. APS层安全性](#62-aps层安全性)
+    - [6.2.1. 总览](#621-总览)
+    - [6.2.2. 安装代码](#622-安装代码)
+- [7. 加入过程](#7-加入过程)
+  - [7.1. 形成网络](#71-形成网络)
+  - [7.2. 使用“已知链接”密钥加入](#72-使用“已知链接”密钥加入)
+  - [7.3. 使用安装代码派生的链接密钥加入](#73-使用安装代码派生的链接密钥加入)
+- [8. 参考](#8-参考)
+</details>
+
+********
+
+## 1. 概述
+### 1.1. 什么是Zigbee
+随着物联网（IoT）行业的发展，越来越多的无线技术应运而生。  
+
+![zigbee](files/ZB-Zigbee-Introduction-of-Zigbee-Basic/IoT-Wireless-Technologies.png)
+
+众所周知，在物联网行业中，我们有两种典型的网络。一个是WAN（广域网），另一个是PAN（个人区域网）。
+
+- 对于LoRa，NB-IoT，2G / 3G / 4G等无线技术，通常传输距离超过1 km，因此它们主要用于广域网（WAN）。
+- 对于WiFi，蓝牙，BLE，Zigbee和Zwave等无线技术，通常的传输距离小于1公里，因此它们主要用于个人局域网（PAN）。
+
+Zigbee是IoT网络（尤其是家庭自动化行业）中最流行的无线技术之一。其特点包括：
+
+- 短距离 – 无线覆盖范围在10到100米之间。
+- 低数据速率 – 最大数据速率为250 Kbps。
+- 低功耗 – 处于睡眠状态的终端设备在睡眠模式下可以使用低于5uA的电流；
+- 这是一种网状技术 – 网络可以轻松扩展到很大。理论上最大节点数为65535。
+
+### 1.2. Zigbee的历史
+Zigbee是[Zigbee联盟](https://zigbeealliance.org/)发布和修订的开放标准。历史悠久。
+
+![zigbee](files/ZB-Zigbee-Introduction-of-Zigbee-Basic/Zigbee-History.png)
+
+从一(很早)开始，Silicon Labs就致力于Zigbee技术。Silicon Labs的Zigbee技术来自Ember，Ember是一家自成立以来就开始研究Zigbee的公司。2012年，Silicon Labs收购了Ember，并继续提供Zigbee产品和解决方案。
+
+### 1.3. Zigbee联盟
+Zigbee联盟是推广Zigbee技术的主要力量。这是一个开放的组织。任何公司都可以加入Zigbee联盟作为成员。Silicon Labs是Zigbee联盟的董事会成员。
+
+![zigbee](files/ZB-Zigbee-Introduction-of-Zigbee-Basic/Zigbee-Alliance.png)
```

Comment:
Please add html control to make both en and cn version image displayed in center. Like this one
```
<div align="center">
  <img src="files/CM-IoT-OTA-Update/ota-diagram.png">
</div> 
```